src/css/profile/mobile/common/dropdownmenu.less

```diff
@@ -256,11 +256,18 @@
 			.font(regular);
 			display: block;
 			position: relative;
-			white-space: nowrap;
 			overflow: hidden;
 			text-overflow: ellipsis;
 			color: var(--dropdown-menu-options-color);
-
+			&.ui-dropdown-two-lines {
+				max-height: 2em;	// FIXME: max-height should be 2 * line-height but gives more than two lines.
```

Comment:
I think it is good to remove the FIXME line. or create an issue and delete it.

and is it correct to use em units?

Reply:
I am not sure if we should create issue for this since visual effect is fine. My intention of adding FIXME is to find better way to display text in two lines (for example, -line-clamp if it's supported in TIZEN).

Using em units is correct here.